<?xml version="1.0"?>

<launch>
	<node name="jir_image_remapper_stereo" type="jir_image_remapper_stereo" pkg="jir_image_remapper_stereo" output="screen" respawn="true">
        <param name="left_map" value="$(find jir_image_remapper_stereo)/../output_data/pinax_maps/left_rect_map.yaml.gz"/>
		<param name="right_map" value="$(find jir_image_remapper_stereo)/../output_data/pinax_maps/right_rect_map.yaml.gz"/>
		<remap from="~in_left" to="/left/image_raw"/>
		<remap from="~out_left_rect/image_raw" to="rectified/left/image"/>
		<remap from="~in_right" to="/right/image_raw"/>
		<remap from="~out_right_rect/image_raw" to="rectified/right/image"/>
	</node>
</launch>
